### PR TITLE
App Crash while creating APKs when template name already exits

### PR DESCRIPTION
**Pull request by sm86**
#338   It is common problem for all the templates and now is fixed.

### CODE REVIEW

File: source-code/app/src/main/java/org/buildmlearn/toolkit/activity/TemplateEditor.java [hunk 1]
```diff
@@ -163,6 +163,9 @@ private void save_apk() {
         String aliasName = getString(R.string.alias_name);
         String aliaspassword = getString(R.string.alias_password);
         KeyStoreDetails keyStoreDetails = new KeyStoreDetails(keyPassword, aliasName, aliaspassword);
+        if(saveProject().equals("File already exists")){
```
Comment (opticod) [inline source-code/app/src/main/java/org/buildmlearn/toolkit/activity/TemplateEditor.java]: You can add all the exceptions from saveProject here. In all those cases, apk should not be created

**Comment by opticod** [comment]
You can add all the exceptions from saveProject here. In all those cases, apk should not be created

**Comment by sm86** [comment]
Thanks @opticod Yeah, I have already added the exception on textfield if same file name exists. Apk doesn't get created in that case.

**Comment by sm86** [comment]
@opticod  waiting for your review

**Comment by opticod** [comment]
@sm86 , Thanks for the PR and your patience. We will review it shortly.

**Comment by opticod** [comment]
@sm86, It looks that you have created the PR targeted to master branch. Can you please redirect to bug-fixes branch.

**Comment by sm86** [comment]
@opticod Am sorry for that, just changed the target branch.

**Pull request merged**